src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -121,12 +128,44 @@ def init_from_file(self) -> None:
 
         self.profiles = profile_jsonc.get("profiles", {})
         self.defaults = profile_jsonc.get("defaults", {})
+        self.schema_property = profile_jsonc.get("$schema", None)
 
         # loading secure props is done in load_profile_properties
         # since we want to try loading secure properties only when
         # we know that the profile has saved properties
         # self.load_secure_props()
 
+    def schema_list(
+        self,
+    ) -> list:
```

Comment:
The `schema_list` method will return the list of `profile_properties` using the `zowe.schema.json` file